federation/backends/collections.md

```diff
@@ -1,6 +1,40 @@
 # Collections
 
-If back-ends offer/mirror the same datasets, it is required to align the metadata.
+If back-ends offer/mirror the same datasets, it is required to align names and metadata.
+
+The following standardized names have been agreed upon:
+
+- SENTINEL2_L1C
+- SENTINEL2_L2A
```

Comment:
What about linking here to the ## Sentinel L2A chapter

Reply:
link added

Reply:
Thx